### ISSUE TITLE
Inline image view

### ISSUE THREAD

**Issue by breadguyyy** [MbinOrg/mbin#1127]
id like the ability to tap an image inline in the feed to open it in full without having to open the post first

**Comment by BentiGorlich** [comment]
I understand your problem, but please use the issue templates in the future.

I've actually wanted to change it for ages 😅
Now I have a good reason

**Comment by breadguyyy** [comment]
lol yeah sorry I was preddy drunk and the issue template looked redundant for an issue this simple to explain. will use the template in the future if need be 😁

**Comment by BentiGorlich** [comment]
And just FYI you can press the little icon next to the "ALT" text:
![grafik](https://github.com/user-attachments/assets/7b88a237-71ae-41c3-90bb-5fe2249e3b2b)

That opens up a preview. On mobile I like to turn the setting "Auto media preview" on, so I don't have to click that icon for every image post :D
On desktop it looks bad imo...

**Comment by adityabad** [comment]
Hello @breadguyyy  . I want to help here. can i take it up?

**Comment by melroy89** [comment]
@adityabad yes. You can. Feel free to fork the project. Create a branch on your fork and create a PR.
